src/services/DealService.ts

```diff
@@ -73,6 +73,7 @@ export class DealService {
     hashes.forEach( async (hash:string) => {
       this.dealsObject[hash] = await (this.ipfsService.getObjectFromHash(hash) as Promise<IDealConfig>)
         .then(async (deal: any) => {
+          if (deal.daos === undefined) return {};
```

Comment:
This is rather a quick fix.
Potential issue: Caller place expects a `Deal` object, that has nested fields.

"Clean fix" suggestion: 
1. return undefined here
2. In caller place if-guard for undefined (`if (result === undefined) return`)